static/app/stores/repositoryStore.tsx

```diff
@@ -79,6 +75,10 @@ const storeConfig: Reflux.StoreDefinition & RepositoryStoreInterface = {
   get() {
     return {...this.state};
   },
+
+  getState() {
```

Comment:
super-nit: you can just do `return this.get();` just so if the `get()` changes you won't have to update it in two places